src/lightning/fabric/strategies/xla.py

```diff
@@ -57,6 +58,7 @@ def __init__(
         self._checkpoint_io: Optional[CheckpointIO]
         self._backward_sync_control = None  # XLA synchronizes gradients in the optimizer.step() call
         self._launched = False
+        self._broadcast_master_params = broadcast_master_params
```

Comment:
Name suggestion to avoid "master" in the name:
- broadcast_main_params
- broadcast_rank_0_params
- broadcast_params

Reply:
If we are not going with the same name as xla's, then we could also consider `sync_module_states` to match FSDP

Reply:
Changed to sync_module_states